src/rules/import-alias.ts

```diff
@@ -122,6 +158,13 @@ interface RelativeImportConfig {
     depth: number;
 }
 
+interface RelativeGlobConfig {
+    // TODO: add
+    pattern: string | RegExp;
+    depth: number;
+}
+type RelativeImportConfig = RelativePathConfig | RelativeGlobConfig;
```

Comment:
Hmm, upon re-evaluating this, it might also just be simpler to have two separate params `overridePatterns` and `overridePaths` (names pending) to avoid any ambiguity. That's entirely outside the scope of this PR though since we'll need to continue to support the existing `relativeImportOverride`, so stick with this! Just noting this to myself for the future.

Reply:
yeah i kind of agree